protobuf/syft_proto/execution/v1/plan.proto

```diff
@@ -16,4 +16,5 @@ message Plan {
     repeated string tags = 6;
     string description = 7;
     bytes torchscript = 8;
+    repeated type serialized_input = 9;
```

Comment:
When I try to compile the stubs, I get:
```
syft_proto/execution/v1/plan.proto:19:14:field syft_proto.execution.v1.Plan.serialized_input: unknown type type
```
I think we'll either have to figure out what data type to use here or create a new Protobuf message type.